server/src/common.ts

```diff
@@ -64,13 +65,14 @@ class Config implements IConfig.Main {
     public server: IConfig.Server = {
         isProduction: false,
         port: 3000,
+        wsPort: 3001,
         cookieMaxAge: 1000 * 60 * 60 * 24 * 30 * 6, // 6 months
         cookieSecureOnly: false,
         postgresURL: "postgresql://localhost/bolt",
         defaultTimezone: "America/New_York"
     };
     public admins = {
-        domains: ["hack.gt"],
+        domains: [] as string[],
```

Comment:
env var this?

Reply:
These are defaults; you can already set them through env vars